components/ai_chat/resources/page/components/site_title/style.module.scss

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
+
+.box {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 16px;
+  padding: 14px 0;
+}
+
+.titleBox {
+  width: 100%;
```

Comment:
why is this needed? Isn't `width: 100%` the default?

Reply:
was a left over probably but i have now added line clamping